SingularityService/src/main/java/com/hubspot/singularity/config/UIConfiguration.java

```diff
@@ -63,6 +64,8 @@ public String getFinishedTaskLogPath() {
   private boolean hideNewDeployButton = false;
   private boolean hideNewRequestButton = false;
 
+  private Optional<Integer> warnIfScheduledJobIsRunningPastNextRunPct;
```

Comment:
i don't think this is necessary -- using the value inside SingularityConfiguration is probably good enough